main.py

```diff
@@ -94,7 +101,7 @@ def loop(self):
         while True:
             frame = self.display.get_frame()
             if frame is None:
-                print(colored("Couldn't read from camera", 'red'))
+                print(colored("couldn't read from camera", 'red'))

```

Comment:
Use single quotes instead of double quotes.
```suggestion
                print(colored('couldn't read from camera', 'red'))
```